llvm/include/llvm/MC/DXContainerPSVInfo.h

```diff
@@ -46,7 +46,7 @@ struct PSVSignatureElement {
 // RuntimeInfo.
 struct PSVRuntimeInfo {
   bool IsFinalized = false;
-  dxbc::PSV::v2::RuntimeInfo BaseData;
+  dxbc::PSV::v3::RuntimeInfo BaseData;
```

Comment:
Shall we add the [PSVStringTable](https://github.com/microsoft/DirectXShaderCompiler/blob/main/include/dxc/DxilContainer/DxilPipelineStateValidation.h#L484) too?
Since the EntryFunctionName in v3::RuntimeInfo is index to the StringTable.

Reply:
I'll look more into this.  It would make sense to have the string table too.  I need to understand what is involved in adding this.

Reply:
The string table already exists, but it is generated in the `write` function in the cpp file.

Reply:
Good to know!